pkg/pillar/hypervisor/kvm.go

```diff
@@ -109,8 +109,8 @@ const qemuConfTemplate = `# This file is automatically generated by domainmgr
   driver = "intel-iommu"
   caching-mode = "on"
 {{ end }}
-[realtime]
-  mlock = "off"
+[overcommit]
+  mem-lock = "off"
```

Comment:
Don't we want --overcommit mem-lock=on
for realtime @rouming ?

Reply:
We don't. On host we don't support memory over-commitment (basically swap is disabled), so we don't need to lock guest memory in order to keep it always available (no swap, no lock is needed). Default setting is ON, but since we don't swap, this basically is NOP, so keeping that explicit is fine to me.